### ISSUE TITLE
Crowd working-platform worker amount

### ISSUE THREAD

**Issue by LeanderK** [coolcrowd/object-service#174]
We have the following problem:
Lets assume we publish an experiment with 1 answer and 1 rating.
If now in the first second 100 workers access the experiment, 100 answers get iussed. This is bad and should not be possible, but there are no easy solutions.
We can not reject the workers on mturk:
`If a Requester is not reasonably satisfied with the Services, the Requester may reject the Services.` (https://requester.mturk.com/policies/conditionsofuse) I don't think that covers this use case, since we don't collect work.
We cannot give 1 answer and 99 ratings since the answer is not existing yet.

Pybossa should not be a problem, since there is no payment-system (and a quick look on crowd crafting did not indicate a problem, since nobody expects a payment there).

The only possible solution is that we publish for the minimum amount of workers able to complete the task and the increase the limit.

**Comment by LeanderK** [comment]
@marcelhollerbach @phimonk

**Comment by marcelhollerbach** [comment]
there is even no solution on mturk, I can pass a maximum number of assignments, the doc for them says 

```
The number of times the HIT can be accepted and completed before the HIT becomes unavailable. 
```

I am not sure if this means that 1 worker acceptes and now the HIT is not visible to anyone.
If we are having 99 acceptest before one worker is completing the hit each one can sent in there answers.

**Comment by marcelhollerbach** [comment]
Okay after a livetest, the doc is wrong, which behaviour should we rely on ?

**Comment by LeanderK** [comment]
i thing the doc is just very vague and tries to describe the behaviour

**Comment by marcelhollerbach** [comment]
yes, but thats a problem, since its the only way to know if this behaviour is a sideeffect or intended

**Comment by LeanderK** [comment]
it is intended behaviour, i asked the mturk support: `In your example, if you limit a HIT to 1 Worker/unique response you will be creating a HIT with 1 assignment for only 1 Worker to complete, no other Workers will have the option to accept it.`

**Comment by marcelhollerbach** [comment]
sweet. still wondering why they dont clarigy such things in the doc...

**Comment by LeanderK** [comment]
will this happen or not? If this will happen i can force the worker for submission of work in the ws

**Comment by simonkorz** [comment]
For pybossa there is no way to control how many workers request a task from the ws. So your scenario is not unlikely (depending on the worker amount very likely) to happen.

**Comment by LeanderK** [comment]
@phimonk not everything is about pybossa ;) this is about mTurk right now

**Comment by simonkorz** [comment]
pybossa, pybossa! :trollface:

**Comment by marcelhollerbach** [comment]
@LeanderK can you rephrase, I dont understand what you mean ?

**Comment by TheRittler** [comment]
Has this been solved?

**Comment by marcelhollerbach** [comment]
No.
